server/aws-lsp-codewhisperer/src/language-server/agenticChat/context/addtionalContextProvider.ts

```diff
@@ -135,24 +140,16 @@ export class AdditionalContextProvider {
         return fileList
     }
 
-    mapToContextCommandItems(context: any[], workspaceFolderPath: string): ContextCommandItem[] {
+    mapToContextCommandItems(context: ContextCommand[], workspaceFolderPath: string): ContextCommandItem[] {
         const contextCommands: ContextCommandItem[] = []
         for (const item of context) {
-            let itemType: ContextCommandItemType | undefined
-            if (item.icon === 'file') {
-                itemType = 'file'
-            } else if (item.icon === 'folder') {
-                itemType = 'folder'
-            } else if (item.icon === 'code-block') {
-                itemType = 'code'
-            }
-            if (itemType) {
+            if (item.route && item.route.length === 2) {
```

Comment:
Is there any reason we skip items that do not have route? What about supporting @sage and @workspace?

Reply:
yes this is intentional, for @workspace we don't need to call the local context library to get additional prompt, we should be able to just store the relevant document when we query them and map that directly to the context list